src/main/java/frc/robot/commands/DriveDistanceGyroPID.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.Constants;
+import frc.robot.subsystems.ChassisSubsystem;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class DriveDistanceGyroPID extends CommandBase {
+  private final ChassisSubsystem m_drive;
```

Comment:
please use camelCase variables. I will be checking my pull request with camelCase variables for consistency.